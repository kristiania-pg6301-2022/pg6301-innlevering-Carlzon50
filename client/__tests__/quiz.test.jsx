import ReactDOM from "react-dom";
import React from "react";
import { FrontPage } from "../Quiz.jsx";
import { MemoryRouter } from "react-router-dom";

describe("quiz application", () => {
    it("shows that front page header is right", () => {
        const element = document.createElement("div");
        ReactDOM.render(
            <MemoryRouter>
                {" "}
                <FrontPage />
            </MemoryRouter>,
            element
        );
        // tester h1
        expect(element.querySelector("h1").innerHTML).toEqual("Quiz App");
        // tester snapshots av diven
        expect(element.innerHTML).toMatchSnapshot();
    });
});