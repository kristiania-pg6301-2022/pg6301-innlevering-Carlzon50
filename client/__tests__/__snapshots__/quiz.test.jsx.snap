// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`quiz application Shows frontpage header is right 1`] = `"<div><h1>Quiz App</h1><ul><li><a href=\\"/question\\">Start Your Quiz</a></li></ul></div>"`;

exports[`quiz application shows that front page header is right 1`] = `" <div><h1>Quiz App</h1><ul><li><a href=\\"/question\\">Start Your Quiz</a></li></ul></div>"`;
